content/200-concepts/150-data-platform/08-data-proxy.mdx

```diff
@@ -85,14 +78,56 @@ datasource db {
 }
 ```
 
-#### Step 5: Generate the client
+##### Vercel Edge
+
+If you want to use Vercel Edge, then
+**!!! (inject the environment variables) - explanation to go here!!!**
+
+##### Cloudflare Workers
+
+If you want to use CloudFlare Workers then
+**!!!! (use global variables) - explanation to go here!!!**
 
-To generate the client using the Data Proxy, set the environment variable `PRISMA_CLIENT_ENGINE_TYPE` to `dataproxy` using the following command:
+#### Step 4: Generate the client
+
+To generate the client with the Data Proxy, call `prisma generate` with the option `--data=proxy`:
 
 ```terminal
-PRISMA_CLIENT_ENGINE_TYPE='dataproxy' npx prisma generate
+prisma generate --data proxy
 ```
 
+The Data Proxy does not need binaries. To keep Prisma Client lightweight, when you use this command the [query engine](/concepts/components/prisma-engines/query-engine) does not copy any binaries to Prisma Client.
```

Comment:
```suggestion
The Data Proxy does not need local engine files. To keep Prisma Client lightweight, when you use this command the [Query Engine](/concepts/components/prisma-engines/query-engine) is not included in the generated Prisma Client.
```

Reply:
@millsp @janpio  To avoid passive, voice, I'd like to say which module doesn't include the query engine in the generated Prisma Client. So I suppose my question is: which Prisma module or component usually includes the query engine in the Prisma Client?

Reply:
If you want to avoid passive voice, you can say something like "The generation process or `prisma generate` does not include the Query Engine in the generated Prisma Client".

Reply:
Ah OK, thanks. That's a bit awkward to state ("Prisma Client does not do X in the generated Prisma Client"), so I will say the following. What do you think?

The Data Proxy does not need local engine files. To keep Prisma Client lightweight, when you use this command the generated Prisma Client does not include the [query engine](/concepts/components/prisma-engines/query-engine).

Note that "query engine" is lower case in the query engine doc, so for now, I've gone with that to be consistent. We are working on terminology standardization in the docs, and deciding capitalization for modules and products is part of that.